state_machine/drivers/emulation/lib/pycubed.py

```diff
@@ -43,26 +44,35 @@ def __init__(self):
         self.c_state_err = 0
         self.c_boot = None
 
+        self._accel = (1.0, 2.0, 3.0)
+        self._mag = (4.0, 3.0, 1.0)
+        self._gyro = (0.0, 0.0, 0.0)
+        self._torque = [0, 0, 0]
```

Comment:
Why is this a list of ints and not a tuple of floats like the others?

Reply:
Doesn't actually make a difference, I set them all to lists because they become lists if values are read anyways.